executor/include/execution/trait/invoke.hpp

```diff
@@ -0,0 +1,266 @@
+/*******************************************************************************
```

Comment:
What do you require from this? Most of the features are present in either Boost or coming soon in PCL (see my PR about lambda filter)

Reply:
I will take a look at that. You are right this is not required anymore, I can now just do `std::forward<F>(f)()`.